ufo2fontir/src/toir.rs

```diff
@@ -73,9 +73,16 @@ pub fn to_ir_axis(axis: &designspace::Axis) -> ir::Axis {
             .iter()
             .map(|map| (UserCoord::new(map.input), DesignCoord::new(map.output)))
             .collect();
+        // # mappings is generally small, repeated linear probing is fine
         let default_idx = examples.iter().position(|(u, _)| *u == default).expect(
             "We currently require that you have a mapping for the default if you have mappings",
         );
+        examples.iter().position(|(u, _)| *u == min).expect(
```

Comment:
I think you need to check not just that min/max are present but that they are in fact the minimum and the maximum.
Unless CoordConverter sorts.. Or is it the PiecewiseLinearMap that guarantee that the array is sorted?

Both the `from` and the `to` coordinate must be increasing, the fromCoord actually has to be _strictly_ increasing. See discussion in avar spec under 'AxisValueMap record'